src/components/informationCard/InformationCard.css

```diff
@@ -0,0 +1,32 @@
+.infoCard {
+    flex: 0 1 calc(50% - 30px);
+    background-color: var(--color-background-light);
+    border-radius: 20px;
+    border: none;
+    padding: 30px;
+    position: relative;
+}
+
+.info-card-title {
+    font-size: 35px;
```

Comment:
Als je het nou helemaal cool wil doen, kun je dit in `rem` opschrijven. In `index.css` staat de font-size in de `html`-selector ingesteld op 14px. Dat betekend dat 1rem gelijk staat aan 14pixels. 35 pixels zou in dit geval `2.5rem` zijn!